pcmsolver/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, cmake, perl, gfortran, python
+# Dependencies
+, boost
+, eigen
+, zlib
+}:
+stdenv.mkDerivation rec {
+    pname = "pcmsolver";
+    version = "1.3.0";
+
+    nativeBuildInputs = [
+      cmake
+      gfortran
+      python
+      perl
+    ];
+
+    buildInputs = [
+      boost
+      eigen
+      zlib
+    ];
+
+    src = fetchFromGitHub  {
+      owner = "PCMSolver";
+      repo = pname;
+      rev = "v${version}";
+      sha256= "117f9ic41xagyzkfwa1sbfcmycc8nlm6ivflkdc228xmp7axvf9m";
```

Comment:
```
unpacking source archive /build/v1.3.0.tar.gz
hash mismatch in fixed-output derivation '/nix/store/16z3y6fdr4sdbrg3m7h5y3hqmiahygxq-source':
  wanted: sha256:117f9ic41xagyzkfwa1sbfcmycc8nlm6ivflkdc228xmp7axvf9m
  got:    sha256:0jrxr8z21hjy7ik999hna9rdqy221kbkl3qkb06xw7g80rc9x9yr
```

Reply:
same as with libefp. I am sorry. It didn't show up anymore as soon as I had it in the nix store and the hash existed somewhere. Checked with `nix-prefetch-url` and should be fixed by [37a969f](https://github.com/markuskowa/NixOS-QChem/pull/6/commits/37a969fa56e77e3e69d5eb37b835383c2644f282).